properties/P000069.md

```diff
@@ -3,8 +3,20 @@ uid: P000069
 name: Strategic Menger
 aliases:
   - M^+
+  - Strategically Menger
+  - \Omega M^+
```

Comment:
Throughout: wrap math notation aliases in `$` `$`.

Reply:
Check